ibis/interactive.py

```diff
@@ -2,7 +2,8 @@
 import ibis.examples as ex
 from ibis import deferred as _
 from ibis import selectors as s
+from ibis.expr.operations import udf
```

Comment:
Do we also want to expose this as `ibis.udf` so users can access this as `ibis.udf.scalar`?

Reply:
Definitely, I will do that in a follow up.